css/stanford_jumpstart_engineering.css

```diff
@@ -51,4 +51,37 @@
   .style-custom-styles-rich #block-bean-jse-linked-logo-block p a {
     padding-bottom: 10px;
   }
-}
\ No newline at end of file
+}
+
+/* grouped grid layout for people */
+
+.views-grid-four .views-row.views-row-4,
+.views-grid-four .views-row.views-row-8,
+.views-grid-four .views-row.views-row-12,
+.views-grid-four .views-row.views-row-16,
+.views-grid-four .views-row.views-row-20,
+.views-grid-four .views-row.views-row-24,
+.views-grid-four .views-row.views-row-28,
+.views-grid-four .views-row.views-row-32,
+.views-grid-four .views-row.views-row-36,
+.views-grid-four .views-row.views-row-40,
+.views-grid-four .views-row.views-row-44,
+.views-grid-four .views-row.views-row-48,
+.views-grid-four .views-row.views-row-52,
+.views-grid-four .views-row.views-row-56,
+.views-grid-four .views-row.views-row-60,
+.views-grid-four .views-row.views-row-64,
+.views-grid-four .views-row.views-row-68,
+.views-grid-four .views-row.views-row-72,
+.views-grid-four .views-row.views-row-76,
+.views-grid-four .views-row.views-row-80,
+.views-grid-four .views-row.views-row-84,
+.views-grid-four .views-row.views-row-88,
+.views-grid-four .views-row.views-row-92,
+.views-grid-four .views-row.views-row-96,
+.views-grid-four .views-row.views-row-100,
+.views-grid-four .views-row.views-row-104,
+.views-grid-four .views-row.views-row-108,
+.views-grid-four .views-row.views-row-112 {
```

Comment:
Thanks, @sherakama! I'll look into the SASS option. Unfortunately, the CSS nth-child(4n) won't work smoothly, as each grouping in the view calculates slightly differently depending on how many items are in the previously grouping (this was my first hunch, too, but was unable to get it working). I'll continue to poke.